tempesta_fw/t/functional/tests/helpers/apache.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python3
+
+__author__ = 'Tempesta Technologies Inc.'
+__copyright__ = 'Copyright (C) 2016 Tempesta Technologies. (info@natsys-lab.com).'
+__license__ = 'GPL2'
+
+import subprocess
+
+
+def is_apache():
+	try:
+		dist = get_dist()
+		if dist == "debian":
+			p = subprocess.Popen(["apache2", "-v"], 
+					     stdout=subprocess.PIPE)
+		elif dist == "centos":
+			p = subprocess.Popen(["httpd", "-v"], 
+					     stdout=subprocess.PIPE)
+	except OSError as e:
+		if e.errno == 2: 
+			return False
+	out = p.stdout.read()
+	print("out:{}".format(out))
+	if out.find("not found") > 0:
+		return False
+	else:
+		return True
+
+def get_dist():
+	p = subprocess.Popen(["cat", "/etc/os-release"], stdout=subprocess.PIPE)
```

Comment:
Why don't you read the file with plain read operation and use IPC call of `cat` process?